include/deal.II/fe/fe_values.h

```diff
@@ -1663,9 +1663,9 @@ class FEValuesBase : protected FEValuesData<dim,spacedim>,
    *
    * @dealiiRequiresUpdateFlags{update_values}
    */
-  template <class InputVector, typename number>
+  template <class InputVector>
   void get_function_values (const InputVector &fe_function,
-                            std::vector<number> &values) const;
+                            std::vector<typename InputVector::value_type> &values) const;
```

Comment:
Oh, I see that actually a lot of them are. I would approve a patch that removes that second template argument for all of those functions that already have it, as part of the current patch.